.rubocop.yml

```diff
@@ -0,0 +1,56 @@
+Documentation:
+  Enabled: false
+
+Metrics/AbcSize:
+  Max: 15
+
+Metrics/BlockLength:
+  CountComments: false
+  Max: 25
+  Exclude:
+    - config/**/*
+    - spec/**/*
+
+Metrics/BlockNesting:
+  Max: 4
+
+Metrics/ClassLength:
+  CountComments: false
+  Max: 200
+
+Metrics/CyclomaticComplexity:
+  Max: 6
+
+Metrics/MethodLength:
+  CountComments: false
+  Max: 24
+
+Metrics/ModuleLength:
+  CountComments: false
+  Max: 200
+
+Metrics/LineLength:
+  Max: 100
+  AllowURI: true
+  URISchemes:
+    - http
+    - https
+
+Metrics/ParameterLists:
+  Max: 5
+  CountKeywordArgs: true
+
+Rails:
+  Enabled: true
+
+Rails/FilePath:
+  Enabled: false
+
+Rails/SaveBang:
+  Enabled: true
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/ModuleFunction:
+  Enabled: false
```

Comment:
Enforce your IDE to leave empty line at end of files

Reply:
are u talking about this configuration `Layout/TrailingBlankLines` ?

Reply:
I'm talking about your IDE configuration (Sublime, Visual Studio, etc). The last line of each file should be left empty in all files (weird developer convention 🤷‍♀) and you can tell your IDE to do that for you so you can forget about it. 
I'm saying this because of the red arrow on the line 56 that tells you that you deleted the extra line